MAINT-25010 : Update documentation for iOS configuration (#6)
exo-addons/push-notifications: README.adoc

==== iOS

* Go to the Firebase console.
* In the center of the project overview page, click the iOS icon to launch the setup workflow.
* If you've already added an app to your Firebase project, click Add app to display the platform options.

> * Enter your app's bundle ID in the iOS bundle ID field. (Make sure that you enter the ID that your app is actually using.) 

Reviewer: ```suggestion
* Enter your app's bundle ID in the iOS bundle ID field. (Make sure to enter the ID your app is actually using.) 
```
